xilem/examples/elm.rs

```diff
@@ -39,9 +40,9 @@ enum AdaptMessage {
     Nop,
 }
 
-// `adapt()` is the most flexible but also most verbose way to modularize the views by state and action,
-// This is basically a combination of `map_state` and `map_action`, but it also allows to change the `MessageResult` for the parent view
-fn adapt_counter(count: i32) -> impl WidgetView<i32, AdaptMessage> {
```

Comment:
I'm not sure whether `AdaptMessage` should be named differently, but I don't think `MapMessage` or `MapMessageMessage` is a good name...